### PR TITLE
Add configurations

### PR DESCRIPTION
**Pull request by DanielMwendwa**

### CODE REVIEW

File: configuration/coding_plans.py
Comment (IsaackMwenda) [inline configuration/coding_plans.py:157]: @as2388 please check with RDA wether they still want to drop this demog question too

File: configuration/pipeline_config.json
Comment (IsaackMwenda) [inline configuration/pipeline_config.json:12]: @as2388 cc @DanielMwendwa  do we have g-cloud demog archives for previous projects that we should add here or should we use the textit versions.

**Review by IsaackMwenda — APPROVED** [review]

File: configuration/pipeline_config.json
Comment (as2388) [reply configuration/pipeline_config.json:12]: Yes, we basically need to take the configuration from USAID-IBTCI and replace the activation flows there with the ones here (so the configuration file ends up being very long)

File: configuration/coding_plans.py
Comment (as2388) [reply configuration/coding_plans.py:157]: have asked on the channel, will wait for a response before merging

**Review by as2388 — CHANGES_REQUESTED** [review]
Thanks a lot @DanielMwendwa, let me know if you'd like me to address any of the comments here if you'd like to

**Comment by DanielMwendwa** [comment]
> Thanks a lot @DanielMwendwa, let me know if you'd like me to address any of the comments here or if you'd like to



> Thanks a lot @DanielMwendwa, let me know if you'd like me to address any of the comments here or if you'd like to

I would like to address the comments.

File: configuration/pipeline_config.json
Comment (DanielMwendwa) [reply configuration/pipeline_config.json:12]: @as2388 kindly work on that, thanks.

Comment (DanielMwendwa) [reply configuration/pipeline_config.json:12]: thanks, updated.

File: configuration/code_schemes.py
Comment (DanielMwendwa) [reply configuration/code_schemes.py]: Thanks, updated.

File: configuration/pipeline_config.json
Comment (as2388) [inline configuration/pipeline_config.json:66]:
```suggestion
    {"SourceKey": "Rqa_Eu_Pcve_S01E01 (Text) - csap_eu_pcve_s01e01_activation", "PipelineKey": "rqa_s01e01_raw", "IsActivationMessage": true},
```

(rapid pro mangles the case of variable names, so it should look like this instead, which you would have seen if you had tested)

File: configuration/pipeline_config.json
Comment (as2388) [inline configuration/pipeline_config.json:204]: Delayded to 08-29

**Review by as2388 — APPROVED** [review]
Thanks, this looks good with minor tweaks

File: configuration/pipeline_config.json
Comment (DanielMwendwa) [reply configuration/pipeline_config.json:78]: Noted, thanks.

**Pull request merged**
